package.json

```diff
@@ -74,6 +74,7 @@
   ],
   "dependencies": {
     "@americanexpress/env-config-utils": "^2.0.2",
+    "@americanexpress/fetch-enhancers": "^1.0.0-rc1",
```

Comment:
If we're using this with One App, shouldn't we be out of `rc` with `fetch-enhancers`?

Reply:
yeah thats a fair point, i don't see anything changing but can hold off until `1.0.0`

Reply:
updated